src/components/SignUp.js

```diff
@@ -143,9 +174,10 @@ export default function SignUp() {
                 helperText={errors.email?.message}
```

Comment:
Always do a null check of the error fields

```suggestion
                helperText={errors?.email?.message}
```